<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WinRt::Internal::WinRtRunConfigurationWidget</class>
 <widget class="QWidget" name="WinRt::Internal::WinRtRunConfigurationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>248</width>
    <height>81</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="argumentsLabel">
       <property name="text">
        <string>Arguments:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="arguments"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="uninstallAfterStop">
     <property name="text">
      <string>uninstall package after stop</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
